docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -1119,6 +1119,10 @@ To ensure high availability when you have multiple instances of Config Server de
 
 If you use HTTP basic security on your Config Server, it is currently possible to support per-Config Server auth credentials only if you embed the credentials in each URL you specify under the `spring.cloud.config.uri` property. If you use any other kind of security mechanism, you cannot (currently) support per-Config Server authentication and authorization.
 
+=== Configuring Read Timeouts
+
+If you want to configure read timeout, this can be done by using the property `spring.cloud.config.requestReadTimeout`.
```

Comment:
We should reference the canonical format, so `request-read-timeout`.

Reply:
Sure, will make the change in the evening.